Makefile

```diff
@@ -7,10 +7,16 @@ update   :; forge update
 install  :; forge install
 
 # Build & test
-build    :; forge clean && forge build --optimize --optimizer-runs 1000000
-test     :; forge clean && forge test -f ${RPC_URL} --fork-block-number ${BLOCK_NUMBER} --etherscan-api-key ${ETHERSCAN_API_KEY} $(call compute_test_verbosity,${V}) # Usage: make test [optional](V=<{1,2,3,4,5}>)
-match    :; forge clean && forge test -f ${RPC_URL} --fork-block-number ${BLOCK_NUMBER} --etherscan-api-key ${ETHERSCAN_API_KEY} -m ${MATCH} $(call compute_test_verbosity,${V}) # Usage: make match MATCH=<TEST_FUNCTION_NAME> [optional](V=<{1,2,3,4,5}>)
-report   :; forge clean && forge test -f ${RPC_URL} --fork-block-number ${BLOCK_NUMBER} --gas-report | sed -e/╭/\{ -e:1 -en\;b1 -e\} -ed | cat > .gas-report
+build    :; forge clean && forge build
```

Comment:
Pretty sure forge clean happens automatically now

Reply:
Nope. Just tried with `forge build`. It doesn't remove the `/out` directory;.